dmoj/settings.py

```diff
@@ -85,6 +85,7 @@
 DMOJ_USER_DATA_CACHE = ''
 DMOJ_USER_DATA_DOWNLOAD_RATELIMIT = datetime.timedelta(days=1)
 DMOJ_COMMENT_VOTE_HIDE_THRESHOLD = -5
+DMOJ_COMMENT_REPLY_TIMEFRAME = datetime.timedelta(days=1000)
```

Comment:
Maybe 365 is better because it's less arbitrary.

Reply:
i should collect statistics to see if 365 is a good cutoff